lib/housekeeping.js

```diff
@@ -67,6 +82,8 @@ class HouseKeeper {
       let stateInstanceIds = stateInstances.map(x => x.id);
       let stateRequestIds = stateRequests.map(x => x.id);
 
+      sweepVolumes();
```

Comment:
Since this is a promise, we should either store the promise and await it later on, or we should await it now.